Description: disable the termwiz feature
 The termwiz feature is already disabled in the ratatui package itself. 
 For consistency, it is also disabled here
Author: Nadzeya Hutsko <nadzya.info@gmail.com>
Forwarded: not-needed
Last-Update: 2025-09-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -95,7 +95,7 @@
 image-defaults = ["image/default"]
 serde = ["dep:serde"]
 termion = ["ratatui/termion"]
-termwiz = ["ratatui/termwiz"]
+#termwiz = ["ratatui/termwiz"]
 
 [target."cfg(not(windows))".dependencies.rustix]
 version = "^0.38.4"
